index.js

```diff
@@ -0,0 +1,14 @@
+import * as base from './src/module/base.js';
+import './style.css';
+
+const inputBtn = document.querySelector('.list-btn');
+const inputData = document.querySelector('.list-input');
+const appList = document.querySelector('.main-body-ul');
+const clearBtn = document.querySelector('.clear-btn');
+
+base.createDataByEnter(inputData, appList, inputData);
+base.createData(inputData, appList, inputBtn);
+base.removeData(appList);
+base.modifyData(appList);
+base.loadData(appList);
+base.registerClearBtnListener(clearBtn);
```

Comment:
- [ ] Based on your webpack set up this file does not belong in the root path. It should be in the `src` folder. Kindly fix that.

- [ ] Fix the resulting import issues.

  ![fhjw](https://user-images.githubusercontent.com/36542986/202156695-74e1286b-b935-4756-922f-fd582736bb6c.jpg)